camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java

```diff
@@ -392,6 +400,31 @@ public int compare(FileStatus f1, FileStatus f2) {
     }
   }
 
+  private void checkIfTooManySkippedMsg(Counter counter) {
+    if (counter.getDisplayName().equals(EtlRecordReader.MSG_SKIPPED.SCHEMA_NOT_FOUND.toString())) {
+      long maxSkippedMsgDueToSchemaNotFound =
```

Comment:
This parameters is probably a percentage value (e.g. 0.1%) instead of absolute value.